packages/core/useClipboard/index.ts

```diff
@@ -101,7 +132,9 @@ export function useClipboard(options: UseClipboardOptions<MaybeRefOrGetter<strin
     ta.style.opacity = '0'
     document.body.appendChild(ta)
     ta.select()
-    document.execCommand('copy')
+    const copyResult = document.execCommand('copy')
+    if (!copyResult)
+      throw new Error('[useClipboard] Fail to copy with legacy method')
```

Comment:
In VueUse we avoid throwing errors, which increase the bundle size and hard to do i18n.

Instead, we could return a boolean indicating if the copy is supported.